CondFormats/SiPixelObjects/interface/SiPixelGainCalibrationForHLTHost.h

```diff
@@ -0,0 +1,9 @@
+#ifndef CondFormats_SiPixelObjects_SiPixelGainCalibrationForHLTHost_h
+#define CondFormats_SiPixelObjects_SiPixelGainCalibrationForHLTHost_h
+
+#include "DataFormats/Portable/interface/PortableHostCollection.h"
+#include "CondFormats/SiPixelObjects/interface/SiPixelGainCalibrationForHLTLayout.h"
+
+using SiPixelGainCalibrationForHLTHost = PortableHostCollection<SiPixelGainCalibrationForHLTLayout<>>;
```

Comment:
```suggestion
using SiPixelGainCalibrationForHLTHost = PortableHostCollection<SiPixelGainCalibrationForHLTSoA>;
```

In general, I would use the `...SoA` alias everywhere instead of the `...Layout<>` template type.